spec/models/tag_rule_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'spec_helper'
 
 describe TagRule, type: :model do
-  let!(:tag_rule) { create(:tag_rule) }
-
   describe "validations" do
     it "requires a enterprise" do
-      expect(tag_rule).to validate_presence_of :enterprise
+      expect(subject).to validate_presence_of(:enterprise)
```

Comment:
I like this pattern, I think we should look for for places where this can be done. I think there will be quite a few.
I'd even create an issue to get this done in a lot more places!

Reply:
Ah, we're testing a class here, so there's no need to instantiate or persist an object. :+1: